src/app.ts

```diff
@@ -167,6 +167,7 @@ class App {
 		// DIDs API
 		app.post('/did/create', DIDController.createDIDValidator, new DIDController().createDid);
 		app.post('/did/update', DIDController.updateDIDValidator, new DIDController().updateDid);
+		app.post('/did/import', new DIDController().importDid);
```

Comment:
This doesn't look right to me.

Where's the validation here?

I don't see any definition either.

Reply:
Added validators for did/import and key/import. Thanks